bin/oneoff/backfill_data/user_cap_state_columns

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+
+# This script migrates CAP (Child Account Policy) data from properties to separate indexed columns.
+
+require 'ruby-progressbar'
+require_relative '../../../dashboard/config/environment'
+
+# @see https://github.com/code-dot-org/code-dot-org/pull/52244
+CAP_RELEASE_DATE = DateTime.parse('2023-06-09')
```

Comment:
I queried our database, and the oldest `current_sign_in_at` is `2023-06-29 18:23:10.000+00:00` where `child_account_compliance_state` is set, so this date should be fine. 👍